### ISSUE TITLE
Rename Framework Image to Framework Intel Image

### ISSUE THREAD

**Issue by czhang03** [ublue-os/framework#72]
Current image name is misleading, as it seems like this image is suitable for all framework laptops. 
However, this image will degrade battery life of both the recently-released framework 13 AMD and framework 16, because of the use of TLP.

**Comment by castrojo** [comment]
Yeah we're currently trying to figure out how to do the naming moving forward because as of now we may not need to even have a dedicated Framework image due to how well things are running with F39 right now. 

The docs mention the Intel-only part, so perhaps I should add a warning to the readme in the meantime. (or if you want to PR that that'd be awesome too!)

**Comment by castrojo** [comment]
From talking with the team it looks like Bluefin and Bazzite will just support Framework out of the box without the need for a special image, so we'll likely drop this repo/image and just have everyone on a better maintained set of images. 

I'll announce and clean up this week.

**Comment by castrojo** [comment]
https://github.com/ublue-os/framework/issues/73
